src/app/producer-shelf/components/tv2-action-card/tv2-action-card.component.ts

```diff
@@ -1,16 +1,60 @@
-import { ChangeDetectionStrategy, Component, HostBinding, Input } from '@angular/core'
-import { Tv2Action } from '../../../shared/models/tv2-action'
+import { ChangeDetectionStrategy, ChangeDetectorRef, Component, ElementRef, HostBinding, Input, OnInit, ViewChild } from '@angular/core'
+import { Tv2Action, Tv2ActionContentType, Tv2VideoClipAction } from '../../../shared/models/tv2-action'
+import { ConfigurationService } from '../../../shared/services/configuration.service'
+import { StudioConfiguration } from '../../../shared/models/studio-configuration'
+
+const ASPECT_RATIO: number = 1.77777777
+const THUMBNAIL_URL_PART: string = '/media/thumbnail/'
 
 @Component({
   selector: 'sofie-tv2-action-card',
   templateUrl: './tv2-action-card.component.html',
   styleUrls: ['./tv2-action-card.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class Tv2ActionCardComponent {
+export class Tv2ActionCardComponent implements OnInit {
   @Input()
   public action: Tv2Action
 
+  @ViewChild('title')
+  public titleRef: ElementRef
+
+  public videoClipThumbnailUrl: string = ''
```

Comment:
Wouldn't `thumbnailUrl` suffice?

Reply:
I considered it, but since this component shows other Actions than just VideoClipActions and since `videoClipThumbnailUrl` is only used for VideoClips, I decided to include `videoClip` in the name so you wouldn't think that it would also work for a GraphicsAction.